internal/store/store.go

```diff
@@ -151,6 +151,8 @@ func (s *Store) Loop(ctx context.Context) error {
 		case actions := <-s.actionCh:
 			s.stateMu.Lock()
 
+			logCheckpoint := s.state.LogStore.Checkpoint()
```

Comment:
(not fault of changes here, but I do wish this was something like `CurrentCheckpoint()` - I always have to go to look and see if it's just a getter or triggering some sort of action)